File: triathlon/src/main/c#/Project/domain/Event.cs
namespace Ro.Mpp.Domain
{
    public class Event : Entity<int>
    {
        public string Name { get; set; }
        public int Referee { get; set; }

        public Event(int id, string name, int referee) : base(id)
        {
            Name = name;
            Referee = referee;
        }
    }
}

File: triathlon/src/main/java/ro/mpp/domain/Event.java
package ro.mpp.domain;

public class Event extends Entity<Integer>{
    private String name;
    private Integer referee;

    public Event(Integer integer, String name, Integer referee) {
        super(integer);
        this.name = name;
        this.referee = referee;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Integer getReferee() {
        return referee;
    }

    public void setReferee(Integer referee) {
        this.referee = referee;
    }
}
